Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java

```diff
@@ -48,55 +48,61 @@ public static String getEmailAddressAttrString() {
     }
 
     /**
-     * Static factory method to examine a BlackboardArtifact to determine if it
-     * has contents that can be used for Correlation. If so, return a
-     * EamArtifact with a single EamArtifactInstance within. If not, return
-     * null.
+     * Examines an artifact and makes zero to many correlation attribute
+     * instances from its attributes.
      *
-     * @param artifact     BlackboardArtifact to examine
-     * @param checkEnabled If true, only create a CorrelationAttribute if it is
-     *                     enabled
+     * IMPORTANT: The correlation attribute instances are NOT added to the
+     * central repository by this method.
      *
-     * @return List of EamArtifacts
+     * @param artifact An artifact.
+     *
+     * @return A list, possibly empty, of correlation attribute instances for
+     *         the artifact.
      */
-    public static List<CorrelationAttributeInstance> makeInstancesFromBlackboardArtifact(BlackboardArtifact artifact,
-            boolean checkEnabled) {
-        List<CorrelationAttributeInstance> eamArtifacts = new ArrayList<>();
+    public static List<CorrelationAttributeInstance> makeAttrsForArtifact(BlackboardArtifact artifact) {
```

Comment:
Terms Artifact & Attrs are used throughout our code and easy to confuse.  I think explicitly naming the method makeCorrAttrsForArtifact() would be useful to clarify this method makes Correlation Attribute (instances) as opposed to TSK attributes.

Reply:
That's a good idea. I will make the change. Please stand by for a new commit.

Reply:
That's a good idea. I will make the change. Please stand by for a new commit.